dsc/tests/dsc.exit_code.tests.ps1

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'exit code tests' {
+    It 'non-zero exit code in manifest has corresponding message' {
+        $result = dsc resource get -r Test/ExitCode --input "{ exitCode: 8 }" 2>&1
```

Comment:
Please add a test for `dsc resource list` operation.
FYI: it emits WARN instead of ERROR.

Reply:
here? https://github.com/PowerShell/DSC/blob/main/dsc_lib/src/discovery/command_discovery.rs#L224-L238

Reply:
Yes.
I guess this applies only to adapters, because `list` for non-adapted resources is just reading a `manifest.json` file.

Reply:
That makes sense, thanks! Completed the manual verification